### ISSUE TITLE
Hi, when using succession with voiceover, it doesn't properly work. 

### ISSUE THREAD

**Issue by AR-Spectacular** [Samgisaninja/SuccessionRestore#132]
Hi, succession doesn't work with voiceover. So I can navigate the menus like donate, and contact aswell as download the ipsw. But after that, i press use provided ipsw and voiceover gets stuck. Any support would be appreciated. Ps, I've used both the beta and normal version. I do remember it working on 12.4 however.

**Comment by Samgisaninja** [comment]
I believe this is probably because the app crashed. There was some attempt at fixing this in 1.4.15, however if it still won’t work, a workaround is not using a provided IPSW, use the built-in downloader instead.
